backend/runner/runner.go

```diff
@@ -568,3 +603,27 @@ func (s *Service) healthCheck(writer http.ResponseWriter, request *http.Request)
 	}
 	writer.WriteHeader(http.StatusServiceUnavailable)
 }
+
+func (s *Service) startPgProxy(ctx context.Context, module *schema.Module, started chan<- pgproxy.Started) error {
+	logger := log.FromContext(ctx)
+
+	databases := map[string]*schema.Database{}
+	for _, decl := range module.Decls {
```

Comment:
Shouldn't this check the database type? Also if there is no databases should it just return?

Reply:
Yes, it should filter by type! Will fix.

It will need to fail if there is no DB, though. That is the error returned to the runner when connecting to the DB

Reply:
Ah, you mean should we not start the proxy at all if there are no PG databases? Yes, good idea. Will change that.

Reply:
Done!